helm/prometheus-rules/templates/alerting-rules/inhibit.all.rules.yml

```
@@ -63,3 +63,12 @@ spec:
         team: phoenix
         topic: kiam
     {{- end }}
+    - alert: InhibitionVintageClusterReleaseBeforeV19
+      annotations:
+        description: '{{`Cluster ({{ $labels.cluster_id }}) using Release ({{ $labels.release_version }}).`}}'
+      expr: aggregation:giantswarm:cluster_release_version{release_version!~"^(1[9]|[2-9]\\d+|[1-9]\\d{2,})\\.[0-9]{1,2}\\.[0-9]{1,2}.*$"}
```

It should also match versions like `v20.123.0`, versions numbers can have more than 1 or 2 digits  AFAIK

```suggestion
      expr: aggregation:giantswarm:cluster_release_version{release_version!~"^(1[9]|[2-9]\\d+|[1-9]\\d{2,})\\.[0-9]+\\.[0-9]+.*$"}
```